docs/code-samples/getting-started/daml/Scripts/Holding.daml

```diff
@@ -0,0 +1,145 @@
+import Daml.Finance.Interface.Holding.Factory qualified as Holding (F)
+import Daml.Finance.Interface.Instrument.Base.Instrument qualified as Instrument (I)
+import Daml.Finance.Interface.Types.Common.Types (AccountKey, Id(..), InstrumentKey(..))
+
+-- IMPLEMENTATION DEPENDENCIES --
+import Daml.Finance.Account.Account qualified as Account (Factory(..))
+import Daml.Finance.Holding.Fungible qualified as Fungible (Factory(..), Fungible(..))
+import Daml.Finance.Instrument.Token.Instrument (Instrument(..))
+
+import Workflow.CreateAccount qualified as CreateAccount
+import Workflow.CreditAccount qualified as CreditAccount
+
+-- | Helper container used to transfer state from one script to another.
+data HoldingState = HoldingState
+  with
+    alice : Party
+    bank : Party
+    bob : Party
+    public : Party
+    aliceAccount : AccountKey
+    bobAccount : AccountKey
+    cashInstrument : InstrumentKey
+    holdingFactoryCid : ContractId Holding.F
+    aliceCashHoldingCid : ContractId Holding.I
+  deriving (Eq, Show)
+
+-- | Test script that
+-- 1. creates an account for Alice and Bob at the Bank
+-- 2. issues a cash instrument
+-- 3. credits a cash holding to Alice in her bank account
+-- 4. transfers the holding from Alice to Bob
+runHolding : Script HoldingState
+runHolding = do
+
+  -- Allocate parties
+  [alice, bank, bob, public] <- mapA createParty $ ["Alice", "Bank", "Bob", "Public"]
+
+  -- Account Factory (it is used by the bank to create accounts)
+  -- CREATE_ACCOUNT_FACTORY_BEGIN
+  accountFactoryCid <- toInterfaceContractId @Account.F <$> submit bank do
+    createCmd Account.Factory with provider = bank; observers = empty
+  -- CREATE_ACCOUNT_FACTORY_END
+
+  -- Holding Factory (it is used by the bank to create holdings with the desired implementation)
+  -- CREATE_HOLDING_FACTORY_BEGIN
+  holdingFactoryCid <- toInterfaceContractId @Holding.F <$> submit bank do
+    createCmd Fungible.Factory with
+      provider = bank
+      observers = fromList [("PublicObserver", singleton public )]
+  -- CREATE_HOLDING_FACTORY_END
+
+  -- Alice and Bob setup account @Bank
+  -- SETUP_ALICE_ACCOUNT_BEGIN
+  aliceRequestCid <- submit alice do
+    createCmd CreateAccount.Request with owner = alice; custodian = bank
+  aliceAccount <- submit bank do
+    exerciseCmd aliceRequestCid CreateAccount.Accept with
+      label = "Alice@Bank"
+      description = "Account of Alice at Bank"
+      accountFactoryCid = accountFactoryCid
+      holdingFactoryCid = holdingFactoryCid
+      observers = []
+  -- SETUP_ALICE_ACCOUNT_END
+
+  bobRequestCid <- submit bob do createCmd CreateAccount.Request with owner = bob; custodian = bank
+  bobAccount <- submit bank do
+    exerciseCmd bobRequestCid CreateAccount.Accept with
+      label = "Bob@Bank"
+      description = "Account of Bob at Bank"
+      accountFactoryCid = accountFactoryCid
+      holdingFactoryCid = holdingFactoryCid
+      observers = [alice]
```

Comment:
`alice` is required to be an observer on the account in order to preform the transfer?